source/develop/developer-guide/vdsm/hook/hugepages.html.md

```diff
@@ -12,11 +12,11 @@ The hugepages vdsm hook will receive hugepages=512 and will preserve 512 huge pa
 
 This is useful for VMs running large memory loads, so that memory page fragmentation is lowered and the VM can access larger memory chunks at a time. The downside is more potential for wasted RAM.
 
-How this works: The hook will add pages: sysctl vm.nr_hugepages=512 and will add add the following xml in domain\\devices:
+How this works: The hook will add pages: sysctl vm.nr_hugepages=512 and will add the following xml in domain\\devices:
 
-`   `<memoryBacking>
-`       `<hugepages/>
-`   `</memoryBacking>
+`   <memoryBacking>  `
```

Comment:
The change doesn't look right:
   <memoryBacking>      <hugepages/>     </memoryBacking>

It probably ought to be in separate lines.